builtin/providers/openstack/resource_openstack_image_v2.go

```diff
@@ -0,0 +1,494 @@
+package openstack
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/imageservice/v2/imagedata"
+	"github.com/gophercloud/gophercloud/openstack/imageservice/v2/images"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
nit: Check the other resource for how imports are ordered. Standard library imports are ordered first, then a blank line, then github/gophercloud, blank link, then github/hashicorp